Formula/spirv-headers.rb

```diff
@@ -0,0 +1,21 @@
+class SpirvHeaders < Formula
+  desc "Headers for SPIR-V"
+  homepage "https://github.com/KhronosGroup/SPIRV-Headers"
+  url "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.3.204.1.tar.gz"
+  sha256 "262864053968c217d45b24b89044a7736a32361894743dd6cfe788df258c746c"
+  license "MIT"
+
+  depends_on "cmake" => [:build, :test]
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    pkgshare.install "example"
+  end
+
+  test do
+    system "cmake", "-S", pkgshare/"example", "-B", "build", *std_cmake_args
```

Comment:
Shouldn't we try to at least build this? Otherwise this seems more like a test of `CMakeLists.txt` in the example rather than a test of the formula.

Reply:
Whoops, that was missed by accident. But there is nothing to run, it's library `libSPIRV-Headers-example.a`.

Reply:
I don't get a justification for such effort, you are talking about writing a test program that will test an example library for headers-only package. I think that compilation check is enough.